docs/grafana.md

```
@@ -0,0 +1,26 @@
+Deploy a new instance of Grafana by clicking the `+ Create` button.
+Enable SMTP if you'd like to receive alerts via email.
+
+
+## Setup Prometheus Data Source
+
+Once the deployment is ready and RUNNING, click on the instance and go to the Connection tab.
+There you can copy the Global endpoint and port, to access the UI.
+Once you log in using the credentials setup in the previous step, you can add a new Prometheus data source by going to Connections > Data Sources > Add Data Source.
+
+Fill in the following field:
+Host: http://prometheus-operated.observability-ns.svc.cluster.local:9090
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)